### ISSUE TITLE
cleaning codes

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#282]
For example:

```bash
./Traj_Tinker.h:11:15: warning: private field 'tinkerTop_' is not used [-Wunused-private-field]
    Topology* tinkerTop_;
              ^
1 warning generated.
```

```bash
Traj_CharmmCor.cpp:18:28: warning: more '%' conversions than data arguments [-Wformat]
      if (sscanf(ptr, "%i %i", ibuf) == 1)
                          ~^
1 warning generated.
```

may be cleaning code after AMBER16?

**Comment by hainm** [comment]
clang is very helpful. It's kind of scary when reading recent email in amberlist (eirdness in gb_ene_hybrid.F90) :D

**Comment by drroe** [comment]
Addressed by #284
